spec/integration/driver_spec.rb

```diff
@@ -56,6 +56,29 @@ def session_url(path)
       end
     end
 
+    unless Capybara::Poltergeist.jruby?
```

Comment:
Hmm, looks like it doesn't actually fail in jruby. I'd be surprised if the thread-safety issue didn't exist there, though. Let me see if I can come up with a better test.